src/styles/stars_styles.scss

```
@@ -0,0 +1,20 @@
+/* #region stars */
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+/* #endregion */
```

I think these comments are not needed here.

```suggestion
.stars {
  display: flex;

  &__star {
    width: 16px;
    height: 16px;
    background-image: url(../images/star.svg);
    background-repeat: no-repeat;
    background-position: center;
    margin-right: 4px;
  }

  &__star:nth-child(-n + 4) {
    background-image: url(../images/star-active.svg);
  }
}
```